gulp-istanbul/gulp-istanbul.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for gulp-istanbul
+// Project: https://github.com/SBoudrias/gulp-istanbul
+// Definitions by: Asana <https://asana.com>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts"/>
+
+declare module "gulp-istanbul" {
+    function GulpIstanbul(opts?: GulpIstanbul.Options): NodeJS.ReadWriteStream;
+
+    module GulpIstanbul {
```

Comment:
I'm wondering if maybe this would work:

```ts
interface GulpIstanbul {
    (opts?: Options): NodeJS.ReadWriteStream;
    hookRequire(): NodeJS.ReadWriteStream;
    summarizeCoverage(...): Coverage;
    writeReports(...): NodeJS.ReadWriteStream;
}
```

It's confusing that `GulpIstanbul` is both a function and a module :S